docs/internals/contributing/bugs-and-features.txt

```diff
@@ -71,7 +71,7 @@ are a few additional guidelines to follow:
   capturing a *brief* screencast. If your software permits it, capture only
   the relevant area of the screen.
 
-* If you're offering a patch that changes the look or behavior of Django's
+* If you're proposing a change that changes the look or behavior of Django's
```

Comment:
+1 for the new wording but with repeating "change" word, the rest of the sentence feels weird.

Reply:
```suggestion
* If you're proposing a change that impacts the look or behavior of Django's
```

Reply:
Well spotted, thanks 👍🏻